Dockerfile

```diff
@@ -197,25 +196,16 @@ RUN pip install yq &&\
 
 RUN apt install -y libffi-dev libssl-dev build-essential python-dev libcurl4-openssl-dev
 
-#COPY scripts/install-dd-plugin-deps.sh /opt/
-
-#RUN mkdir -p /opt/dd &&\
-    #cd /opt/dd &&\
-    #git clone --depth 1 --single-branch https://github.com/DataDog/dd-agent.git &&\
```

Comment:
I noticed neopy/datadog.py is removed w/ no port as well.  Is removing dd support an intended part of this or is it provided through another mechanism?

Reply:
The neopy thing had been broken since the v3 release and nobody is using it (it was my hackathon project last year).  It would be pretty easy to reenable if desired though.